src/components/ReposPopup.vue

```diff
@@ -51,6 +53,9 @@ export default {
     }
   },
   methods: {
+    getBuildActor(build) {
```

Comment:
just `getBuildActor,` should be enough. Like here https://github.com/drone/drone-ui/blob/vue/src/components/AlertError.vue#L19